modules/opm-cli-ref-serve.adoc

```diff
@@ -23,13 +23,28 @@ $ opm serve <source_path> [<flags>]
 |===
 |Flag |Description
 
+|`--cache-dir` (string)
+|If this flag is set, it syncs and persists the server cache directory.
+
+|`--cache-enforce-integrity`
+|Exits with an error, if the cache is not present or is invalidated. The default value is `true` when the `--cache-dir` flag is set and the `--cache-only` flag is `false`. Otherwise, the default is `false`.
+
+|`--cache-only`
+|Syncs the serve cache and exits without serving.
+
 |`--debug`
-|Enable debug logging.
+|Enables debug logging.
+
+|`h`, `--help`
+|Help for serve.
 
 |`-p`, `--port` (string)
-|Port number to serve on. Default: `50051`.
+|The port number to serve on. The default value is `50051`.
+
+|`--pprof-addr` (string)
+|The address of the startup profiling endpoint. Addr: Port format.
```

Comment:
```suggestion
|The address of the startup profiling endpoint. Addr:Port format.
```

Reply:
Suggestion: I believe there is no space adjacent to the ":"

Reply:
Done.